public/index.html

```diff
@@ -10,11 +10,11 @@
     <meta property="og:title" content="Mpls Jr Devs" />
     <meta property="og:description" content="A community of aspiring and less experienced software engineers in Minneapolis/St. Paul" />
     <meta property="og:url" content="https://mplsjrdevs.com/"/>
-    <meta property="og:image:secure_url" content="https://mplsjrdev.com/mplsjrdevs-logo-color.jpg"/> <!-- https -->
-    <meta property="op:image:alt" content="Mpls Jr Devs Logo"/>
+    <meta property="og:image" content="http://mplsjrdevs.com/mplsjrdevs-logo-color.jpg"/>
```

Comment:
you still need the `https` here. I don't think this will resolve

Reply:
Good catch